README.md

```diff
@@ -0,0 +1,25 @@
+# signed_json
+
+An implementation to read signed jsons
+
+## How to use it
+
+This is the library that should be used to verify the files generated by `https://github.com/jeroentrappers/sign_config`
```

Comment:
generated by [sign_config](https://github.com/jeroentrappers/sign_config)

Reply:
maybe also provide more details on how to use? And ask Jeroen to put this on pub.dev?

Reply:
add that you need to check config/remote_config_sign.sh